node/core/pvf/src/worker_common.rs

```diff
@@ -206,64 +206,65 @@ where
 /// background. When it wakes, it will see that the flag has been set and return.
 pub async fn cpu_time_monitor_loop(
 	job_kind: JobKind,
-	mut stream: UnixStream,
+	mutex: Arc<Mutex<(UnixStream, bool)>>,
 	cpu_time_start: ProcessTime,
 	timeout: Duration,
-	lock: Arc<AtomicBool>,
```

Comment:
That this parameter was called "lock" is also not a great naming. I mean I know what it does by reading the code, but this thing should be really better named!